scripts/loki_transform.py

```diff
@@ -38,6 +38,12 @@
 from transformations.single_column_claw import ExtractSCATransformation, CLAWTransformation
 from transformations.single_column_coalesced import SingleColumnCoalescedTransformation
 from transformations.scc_cuf import SccCufTransformation, HoistTemporaryArraysDeviceAllocatableTransformation
+from transformations.scc_base import SCCBaseTransformation
```

Comment:
I've put `SingleColumnCoalescedTransformation` into a separate module `single_column_coalesced_wrapper.py` to prevent a circular import. Since this transformation is earmarked for deletion in the near future anyway, also makes sense to put it in a separate module.